color-spantrace/src/lib.rs

```diff
@@ -60,10 +60,11 @@
 //! [`tracing_error::SpanTrace`]: https://docs.rs/tracing-error/*/tracing_error/struct.SpanTrace.html
 //! [`color-backtrace`]: https://github.com/athre0z/color-backtrace
 #![doc(html_root_url = "https://docs.rs/color-spantrace/0.2.0")]
+#![cfg_attr(feature = "rustdoc_missing_doc_code_examples",
+            warn(rustdoc::missing_doc_code_examples))]
```

Comment:
this doesn't actually work. `feature = "..."` is a cargo thing, which is different from "nightly features" (the thing it was erroring about).

afaik the only way to do this like you expect is with hacks like compiling a file in build.rs to see if it supports nightly features, or by manually making people say `cargo doc --features rustdoc_missing_doc_code_examples`.

you should ask jane if she prefers that latter thing, saying people have to use nightly to document (and adding `#![feature()]`, or just removing the warning altogether.

Reply:
some more context - this is what cargo means by "feature" https://doc.rust-lang.org/cargo/reference/features.html
and this is what rustc means https://doc.rust-lang.org/book/appendix-07-nightly-rust.html

Reply:
oh, one last option is to add this warning in CI only with `-Z crate-attr`, but that would be kind of annoying for people building locally since `cargo doc` won't show as many warnings as CI